configs/a64fx.yaml

```diff
@@ -272,3 +272,5 @@ CPU-Info:
   # Package-Count is used to generate
   # /sys/devices/system/cpu/cpu{0..Core-Count}/topology/{physical_package_id, core_id}
   Package-Count: 1
+Environment-Variables:
+  - "OMP_NUM_THREADS=2"
```

Comment:
What has this env variable been set to 2

Reply:
changed to 1